helmfile/values/user-rbac.yaml.gotmpl

```diff
@@ -2,4 +2,6 @@ namespaces: {{ toYaml .Values.user.namespaces | nindent 2 }}
 users: {{ toYaml .Values.user.adminUsers | nindent 2 }}
 createNamespaces: {{ .Values.user.createNamespaces }}
 enableFalcoViewer: {{ toYaml .Values.falco.enabled | nindent 2 }}
-alertmanagerNamespace: {{ .Values.user.alertmanager.namespace }}
+alertmanager:
+  enabled: {{ .Values.user.alertmanager.enabled }}
+  namespace: {{ .Values.user.alertmanager.namespace }}
```

Comment:
We need a migration for this, right? Translating the old `alermanagerNamespace` to `alertmanager.namespace`.

Reply:
I don't think so, the namespace is still assumed from the same template variable: `{{ .Values.user.alertmanager.namespace }}`.

Reply:
Ah sorry, I read that a bit too fast. :sweat_smile: